### DISCUSSION TITLE
Navigate site using shortcuts

### DISCUSSION THREAD

**Discussion by pingshunhuangalex** [monkeytypegame/monkeytype#1215] [Ideas] [answered]
It's a bit annoying that I need to use a mouse to click on tabs like account and settings etc. It'll be nice if I can quickly go to a different page using a quick shortcut. Ideally, I want to navigate to any part of the site using the keyboard and not touching the mouse at all. Thanks.

**Comment by unrealapex** [comment, marked as answer]
This feature is already implemented through the command line, type `View... `.

**Reply by Miodec** [reply]
You can already change pages with the command line

**Reply by pingshunhuangalex** [reply]
I was more talking about navigation using a single keystroke cuz you can always press `Esc` to start a new test wherever I want. I'm looking for a similar behaviour.

**Reply by unrealapex** [reply]
Switching from the typing page to any other page wouldn't be possible because every other key besides `esc`, `ctrl`,`tab`, `space`, `caps lock`, `backspace`,or `shift` would trigger a test start. Furthermore, I don't think it would be that useful because all commands can already be accessed via the command line.
